frontend/lib/queries/autogen/DDOSuggestionsResult.graphql

```diff
@@ -18,7 +18,7 @@ fragment DDOSuggestionsResult on DDOSuggestionsResult {
   portfolioTopBorough,
   numberOfBldgsInPortfolioTopBorough,
   stabilizedUnitCount2007,
-  stabilizedUnitCount2017,
+  stabilizedUnitCount2019,
```

Comment:
Uh-oh, changing the name of the GraphQL field from `stabilizedUnitCount2017` to `stabilizedUnitCount2019` isn't good because out-of-date front-ends will get an error from the server when they try to fetch `stabilizedUnitCount2017` 😞.  Is it possible to deprecate `stabilizedUnitCount2017` and instead add _two_ new GraphQL fields, one called `stabilizedUnitCount` and `stabilizedUnitCountYear`, and have the latter just be hard-coded to return "2019" (it could even be part of the SQL)?  That way in the future this kind of change need consist _only_ of back-end changes, and no alteration in the GraphQL schema or front-end code!